src/main/java/com/automattic/elasticsearch/plugin/StatsdPlugin.java

```diff
@@ -40,13 +41,14 @@
                 REPORT_FS_DETAILS_S,
                 NODE_NAME_S,
                 PREFIX_S,
-                TEST_MODE_S
+                TEST_MODE_S,
+                INDEXING_PRESSURE_S
```

Comment:
You'll also need to modify [StatsdReporterNodeStats.run](https://github.com/Automattic/elasticsearch-statsd-plugin/blob/master/src/main/java/com/automattic/elasticsearch/statsd/StatsdReporterNodeStats.java#L24) to send indexing pressure stats.

Also, it would be good to test the new stats end to end locally by deploying an ES and Graphite instance.